lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -307,3 +307,70 @@ def upload_ora2_data(
     TASK_LOG.info(u'%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def upload_ora2_summary(
+        _xmodule_instance_args, _entry_id, course_id, _task_input, action_name
+):
+    """
+    Collect ora2/student summaries and upload them to S3 as a CSV
```

Comment:
also note that this isn't always S3 -- the report store is configurable to any django storage backend.